Automattic/studio
# Studio: Display spinner when the site is being pulled

## Related issues

- Fixes https://github.com/Automattic/dotcom-forge/issues/9897.

## Proposed Changes

- display spinner when the site is being pulled, use `Syncing` tooltip
- add tooltip messages for other cases where the spinner is displayed:
  - `Adding` - when a new site is being added
  - `Importing` - when an exported site is being imported
  - `Loading` - fallback tooltip message 

![Markup on 2024-11-19 at 17:37:43](https://github.com/user-attachments/assets/3dccfe8f-b539-482a-aa77-d782aa8e1496)

## Testing Instructions



1. Check out the PR and build the app with `STUDIO_SITE_SYNC=true npm start`.
2. Open the Sync tab.
3. Connect a WP.com site to Studio.
4. Pull the site from WP.com.
5. Observe the spinner by the site in the sidebar. 

Testing other tooltip messages:

1. Go through the process of adding a new site and observe the `Adding` tooltip when the site is being added.
2. Import an exported site and observe the `Importing` tooltip as the site is being imported.

## Pre-merge Checklist
